### PR TITLE
fix(dedicated): add back order retractation screen

### PR DESCRIPTION
**Pull request by frenautvh**
### Description of the Change

ref: DTRSD-6243, DTRSD-6375

44e9cc1fd8 — fix(dedicated): add back order retractation screen

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/billing/orders/billing-orders.controller.js [hunk 1]
```diff
@@ -26,7 +26,10 @@ export default class BillingOrdersCtrl {
     return this.OvhApiMeOrder.v6()
       .getStatus({ orderId: $row.orderId })
       .$promise
-      .then(status => assign($row, status));
+      .then(status => assign($row, status))
+      .then(() => {
+        assign($row, { canRetract: moment($row.retractionDate || 0).isAfter(this.timeNow) });
+      });
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/billing/orders/billing-orders.controller.js]:
```
 .then(status => assign($row, status, { canRetract: moment($row.retractionDate || 0).isAfter(this.timeNow) }))
```

File: packages/manager/apps/dedicated/client/app/billing/orders/billing-orders.html [hunk 2]
```diff
@@ -77,7 +77,7 @@
             </oui-action-menu-item>
             <oui-action-menu-item
                 data-ng-if="$row.canRetract"
-                data-ui-sref="app.account.billing.orders.retract({ id: $row.orderId })">
+                data-ui-sref="app.account.billing.retract({ id: $row.orderId })">
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/billing/orders/billing-orders.html]: This should have been injected through the routing

File: packages/manager/apps/dedicated/client/app/billing/orders/retraction/billing-orders-retraction.html [hunk 3]
```diff
@@ -1,55 +1,72 @@
...
-           data-translate="orders_retract_return_to_orders">
-        </a>
-    </div>
+<div id="billing-payment-retraction" class="billing-orders">
 
-    <oui-back-button data-href="{{ ctrl.$state.href('^') }}"
-                     data-title="{{:: 'orders_retract_title' | translate }}">
-    </oui-back-button>
+    <oui-page-header data-heading="{{ :: 'orders_page_title' | translate }}">
+        <oui-guide-menu text="{{:: 'orders_guides' | translate }}"
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/billing/orders/retraction/billing-orders-retraction.html]:
```suggestion
        <oui-guide-menu data-text="{{:: 'orders_guides' | translate }}"
```

File: packages/manager/apps/dedicated/client/app/billing/orders/retraction/billing-orders-retraction.html [hunk 4]
```diff
@@ -1,55 +1,72 @@
...
-            </button>
+        </div>
+
+        <div data-ng-if="!$ctrl.success">
+            <div data-ovh-alert></div>
+
+            <p data-translate="orders_retract_description1"></p>
+            <p data-translate="orders_retract_description2"></p>
+
+            <p>{{ 'orders_retract_confirmation' | translate:{ t0: $ctrl.orderId } }}</p>
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/billing/orders/retraction/billing-orders-retraction.html]:
```suggestion
            <p data-translate="orders_retract_confirmation" data-translata-values="{ t0: $ctrl.orderId }"></p>
```

File: packages/manager/apps/dedicated/client/app/billing/orders/retraction/billing-orders-retraction.html [hunk 5]
```diff
@@ -1,55 +1,72 @@
...
+            <div class="checkbox">
+                <label>
+                    <input type="checkbox"
+                           data-ng-model=":: confirmed">
+                    {{ 'orders_retract_confirmation_text' | translate }}
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/billing/orders/retraction/billing-orders-retraction.html]:
```suggestion
                    <span data-translate="orders_retract_confirmation_text"></span>
```

**Comment by cbourgois** [comment]
PR updated and rebased on master.

**Pull request merged**